test/jdk/java/util/ResourceBundle/Control/MissingResourceCauseTestRun.java

```diff
@@ -50,9 +51,14 @@
 import jdk.test.lib.JDKToolLauncher;
 import jdk.test.lib.Utils;
 import jdk.test.lib.process.ProcessTools;
+import jdk.test.lib.Platform;
+import jtreg.SkippedException;
 
 public class MissingResourceCauseTestRun {
     public static void main(String[] args) throws Throwable {
+        if(Platform.isRoot() && !Platform.isWindows()) {
```

Comment:
```suggestion
        if (Platform.isRoot() && !Platform.isWindows()) {
```

Reply:
Thanks for  the review. A white space before `if` has been added.